File: platforms/swift/examples/texture/mipmap/size.swift
import FragmentColor
import Foundation

let pixels = Data(repeating: 0, count: 16 * 16 * 4)
let chain = try Mipmap.build(
    bytes: pixels,
    format: .rgba8UnormSrgb,
    size: Size(width: 16, height: 16, depth: nil)
)
let sz = chain.size()
let width = sz.width
let height = sz.height
let _ = (width, height)

File: platforms/web/examples/texture/mipmap/size.js
import { TextureFormat, Mipmap } from "fragmentcolor";

const pixels = new Uint8Array(16 * 16 * 4);
const chain = Mipmap.build(pixels, TextureFormat.Rgba8UnormSrgb, [16, 16]);
const sz = chain.size();
const width = sz.width;
const height = sz.height;